website/tutorial/part_1/03_installation.md

```diff
@@ -11,68 +11,32 @@
 
 Exosphere is not just a simple application,
 but a framework for large-scale software development of micro-service based applications.
-Developing applications consisting of many different code bases requires:
-* a __package management system__ for installing/updating the various pieces of platforms and tools as needed.
+Developing applications consisting of many different code bases requires
+a __package management system__ for installing/updating the various pieces of platforms and tools as needed.
   You don't want to get into the business of having to install several programming languages,
   frameworks, and other dependencies of your polyglot applications manually.
 
-The installation instructions below include
-making these infrastructure components available on your system.
+* Download and install the binary executable for your platform from the 
+[GitHub release page](https://github.com/Originate/exosphere/releases)
+* Rename the executable to `exo`
+* Check and update your `PATH` environmental variable to include its install location, if required
```

Comment:
I would suggest moving the `exo` binary into a directory that is in the path. This is common for installation instructions.

Reply:
I've reworked the install instructions, but only have specific installation instructions for Mac. I'm not familiar enough with scripting on Windows to write Windows instruction.

Reply:
Don't worry about platforms you are not familiar with. Thanks for doing this for the Mac!